### ISSUE TITLE
More Integrations

### ISSUE THREAD

**Issue by 31453** [arkime/arkime#2130]
- [x]  DomainTools (#3406)
- [x] RDAP for domain and RDAP IP card (#3409)
- [ ] Zetalytics
- [x] CRT.sh (#3410)
- [x] [AWS Pinpoint for phone number](https://docs.aws.amazon.com/pinpoint/latest/apireference/phone-number-validate.html) (won't do because API is being shut down)
- [x] Greynoise: Generate a tidbit when `"classification": "malicious"`. Display a red `malicious` tidbit with content of `${integration_name}: [classification] + ', ' + [name]`. Would also like to make the tidbit clickable to pivot to the greynoise api result [link] value. (#3407)
- [ ] ThreatFox: Add ThreatFox integration to domain iType processing.

**Comment by Avedena** [comment]
Also a nice integration for German Law Enforcement Agency is [https://www.dialogika.de/en/produkte/infreq100/](InfReq100)

**Comment by 31453** [comment]
@WilliamSalusky I'm waiting on a key from Zetalytics and can you please explain ThreatFox? Do you mean you want a ThreatFox Domain Integration? or something else?

@Avedena Can you get me access to that dataset for testing and some API documentation? If so, I can add it! Otherwise I could mock out something and you could finish it up if you have access to test with real data.
